common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -161,6 +161,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-acquisitions-epic-always-ask-election",
+    "This will guarantee that the epic is always displayed on election stories",
+    owners = Seq(Owner.withGithub("jranks123")),
+    safeState = On,
```

Comment:
I have always gone for on, I thought about it and I couldn't figure out any benefit of having it not on...as soon as they test is deployed, I want it to be on. In the past, I've forgotten to turn on tests, and this safe state doesn't mean you can't turn it off if you don't want to. But if there is a good reason for having it off I'd love to hear it...